modules/ROOT/partials/configuration/advanced-typography.adoc

```diff
@@ -0,0 +1,247 @@
+[[content_langs]]
+== `+content_langs+`
```

Comment:
I don't think that we should document this in typography as it's not typography specific.